### PR TITLE
Fix wrong variable, making it impossilbe to activate linting

### PR DESCRIPTION
**Pull request by KingKili**

### CODE REVIEW

**Comment by KingKili** [comment]
I thinkt the CI failing is not my fault

File: CMakeLists.txt [hunk 1]
```diff
@@ -43,7 +43,7 @@ include(sanitizer)
 include(gcc_analyze)
 include(clear_variable)
 
-if (${JUTTA_UI_ENABLE_LINTING})
+if (ENABLE_LINTING)
```
Comment (COM8) [inline CMakeLists.txt]:
```suggestion
if (UWPX_PUSH_SERVER_ENABLE_LINTING)
```

**Review by COM8 — CHANGES_REQUESTED** [review]

**Comment by COM8** [comment]
> 
> 
> I thinkt the CI failing is not my fault

Yes, there is something else broken in Ubuntu with conan.

**Review by COM8 — APPROVED** [review]
Thanks!

**Pull request merged**